back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -207,6 +212,10 @@ def update
     CustomFieldService.new.cleanup_custom_field_values! update_params[:custom_field_values]
     input.assign_attributes update_params
     authorize input
+    if anonymous_not_allowed?(participation_context)
+      render json: { errors: { base: [{ error: :anonymous_participation_not_allowed }] } }, status: :unauthorized
```

Comment:
Slightly in two minds but I think maybe `:unprocessable_entity` might be a better return code

Reply:
I was also hesitating. Fine for me to change it.